AdminDashboard/BusinessLogicOrchestrators/AccountOrchestrator/AccountOrchestrator.cs

```diff
@@ -35,6 +40,8 @@ public async Task RegisterUserToAccount(Guid accountId, Guid userId)
                 throw new UserLimitExceededException();
 
             await Repository.AddUser(accountId, userId);
+
+            await UserHubContext.Clients.All.SendAsync("AddedUser");
```

Comment:
This initiates a server side request to tell the client that another user has been registered